components/contributors/GithubActivity.tsx

```diff
@@ -387,6 +396,11 @@ export default function GithubActivity({ activityData }: Props) {
   };
 
   const [activityTypes, setActivityTypes] = useState([...ACTIVITY_TYPES]);
+  const [storedFilters, setStoredFilters] = useAtom(activityTypesAtom);
```

Comment:
the atom itself can be used to manage the state right?

```suggestion
  const [activityTypes, setActivityTypes] = useAtom(activityTypesAtom);
```

Reply:
Done.